packages/client/src/__tests__/integration/errors/uncheckedScalarValidation/test.ts

```diff
@@ -42,56 +39,38 @@ testIf(getQueryEngineProtocol() !== 'json')('uncheckedScalarInputs validation',
 
-         9 const prisma = new PrismaClient()
-        10 
-        11 try {
-      → 12   await prisma.trip.create({
-               data: {
-                 name: 'Trip 1',
-                 description: 'This is a description',
-                 public: false,
-                 dateFrom: new Date('2020-12-29T10:15:16.495Z'),
-                 dateTo: new Date('2020-12-29T10:15:16.495Z'),
-                 adults: 12,
-                 backgroundUrl: 'https://duckduckgo.com',
-                 userId: 1,
-                 activities: {
-                   create: [
-                     {
-                       name: 'Activity1',
-                       description: 'This is activity 1',
-                       location: 'Some location',
-                       date: new Date('2020-12-29T10:15:16.495Z'),
-                       public: false,
-                       timezone: 'Europe/Berlin',
-                       maxPeople: 1,
-                       activityTypeId: 1,
-                       tripId: 1
-                       ~~~~~~
-                     }
-                   ]
-                 }
-               }
-             })
-
-      Unknown arg \`tripId\` in data.activities.create.0.tripId for type ActivityUncheckedCreateWithoutTripInput. Did you mean \`id\`? Available args:
-      type ActivityUncheckedCreateWithoutTripInput {
-        id?: Int
-        uuid?: String
-        activityTypeId: Int
-        name: String
-        description?: String | Null
-        location: String
-        date: DateTime
-        timezone: String
-        public: Boolean
-        maxPeople?: Int | Null
-        createdAt?: DateTime
-        updatedAt?: DateTime
-        users?: UsersToActivitiesUncheckedCreateNestedManyWithoutActivityInput
-      }
-
+        6 const prisma = new PrismaClient()
+        7 
+        8 try {
+      → 9   await prisma.trip.create({
+              data: {
+                name: "Trip 1",
+                description: "This is a description",
+                public: false,
+                dateFrom: new Date("2020-12-29T10:15:16.495Z"),
+                dateTo: new Date("2020-12-29T10:15:16.495Z"),
+                adults: 12,
+                backgroundUrl: "https://duckduckgo.com",
+                userId: 1,
+                activities: {
+                  create: [
+                    {
+                      name: "Activity1",
+                      description: "This is activity 1",
+                      location: "Some location",
+                      date: new Date("2020-12-29T10:15:16.495Z"),
+                      public: false,
+                      timezone: "Europe/Berlin",
+                      maxPeople: 1,
+                      activityTypeId: 1,
+                      tripId: 1
+                    }
+                  ]
+                }
+              }
+            })
 
+      Argument \`activityType\` is missing.
```

Comment:
Looks like with JSON protocol this input is validated as checked and not unchecked, `activityType` shouldn't be required since `activityTypeId` is specified.